aten/src/ATen/native/TensorCompare.cpp

```diff
@@ -87,6 +87,16 @@ Tensor isnan(const Tensor& self) {
   return self != self;
 }
 
+Tensor isfinite(const Tensor& self) {
+  // Integral tensor types are finite
+  if (!self.is_floating_point()) {
+    return at::ones_like(self, at::kBool, at::MemoryFormat::Preserve);
```

Comment:
This used to be `torch.legacy_contiguous_format`; is this change OK?

Reply:
I discussed it with Vitaly and he suggested to make it `at::MemoryFormat::Preserve`